src/app/(proper_react)/(redesign)/(public)/LandingViewRedesign/components/PrivacyProductBundleBanner.tsx

```diff
@@ -27,6 +29,14 @@ export const PrivacyProductBundleBanner = (props: Props) => {
     currency: "USD",
     currencyDisplay: "narrowSymbol",
   });
+  const searchParams = new URLSearchParams({
```

Comment:
I feel like this is like the fourth time I've seen this pattern, so probably best to keep this to stay consistent, but... You could've saved yourself the need to add a comment and string wrangling using:

```ts
const upgradeUrl = new URL(props.premiumSubscriptionUrlBundle);
const searchParams = new URLSearchParams(upgradeUrl.search);
searchParams.set("utm_medium", "monitor");
searchParams.set("utm_source", "monitor-product");
searchParams.set("utm_campaign", "landing-page-banner");
searchParams.set("utm_content", "banner-us");
updateUrl.search = `?${searchParams.toString()}`;

// ...later
  href={upgradeUrl.toString()}
```